common/djangoapps/third_party_auth/provider.py

```diff
@@ -39,7 +39,7 @@ def enabled(cls):
     @classmethod
     def accepting_logins(cls):
         """Returns list of providers that can be used to initiate logins currently"""
-        return [provider for provider in cls.enabled() if provider.accepts_logins]
+        return [provider for provider in cls.enabled() if provider.display_for_login]
```

Comment:
I think "accepting logins" is semantically different than "displaying for login" -- does it make sense to keep accepting_logins() as-is and introduce a new "displayed_for_logins" classmethod to cover this use case?